src/libs/CurrencyUtils.ts

```diff
@@ -110,11 +111,20 @@ function convertToFrontendAmountAsString(amountAsInt: number | null | undefined)
  * @param amountInCents – should be an integer. Anything after a decimal place will be dropped.
  * @param currency - IOU currency
  */
-function convertToDisplayString(amountInCents = 0, currency: string = CONST.CURRENCY.USD): string {
+function convertToDisplayString(amountInCents = 0, currency: keyof CurrencyList | undefined = CONST.CURRENCY.USD): string {
```

Comment:
https://github.com/Expensify/App/blob/fcfef92de701acca9f4a1d18a8d2e64a356048bb/src/types/onyx/Currency.ts#L24

It seems `keyof CurrencyList ` not work as expected as the list come from backend and save to onyx not static in the code so the `keyof CurrencyList` is `string[]`

I think we need to revert this change.

Reply:
<img width="985" alt="Screenshot 2024-06-06 at 2 30 27 AM" src="https://github.com/Expensify/App/assets/59088937/50a9e62a-ec5c-429a-81fb-d0e13f64fd77">

I don't think so, we can keep it, I don't think there's no issue with this.

Reply:
Yes, there are no issues with it, But it not works as expect. If you pass currency as `''` it not display type error, so we need to revert it to `string`